plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/CassandraQueryRunner.java

```diff
@@ -24,13 +25,20 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import static io.airlift.log.Level.OFF;
 import static io.trino.plugin.cassandra.CassandraTestingUtils.createKeyspace;
 import static io.trino.plugin.tpch.TpchMetadata.TINY_SCHEMA_NAME;
 import static io.trino.testing.QueryAssertions.copyTpchTables;
 import static io.trino.testing.TestingSession.testSessionBuilder;
 
 public final class CassandraQueryRunner
 {
+    static {
+        Logging logging = Logging.initialize();
+        logging.setLevel("com.datastax.oss.driver.internal", OFF);
+        logging.setLevel("io.trino.plugin.cassandra.CassandraServer", OFF);
```

Comment:
This is our own code, so we can make them `log.debug()` instead of disabling logging.